### DISCUSSION TITLE
Different text wraplengths for different columns?

### DISCUSSION THREAD

**Discussion by JustMentalMatt** [Akascape/CTkTable#91] [Q&A]
When entering text into columns, they resize when the text exceeds the cell demensions, so setting expand to False or setting width equal to something doesnt work, as the cell seems to automatically constrain to the size of whatever text is entered.

However, using **wraplength** fixes the length of text before it wraps onto a new line, keeping the width consistent, however i cant seem to get this to work for individual colums, rather it only lets me set wraplength for the entire table.

e.g table.edit_column(2, wraplength=25) doesnt work.

This means that some columns such as FirstName or Username have the same wrap length, however **i would like to have different wraplengths for each column.**

Hopefully that makes some sense, thank you!

**Comment by oplix6417** [comment]
Hey, try setting your width to 0 (width=0) in the CTktable arguments.
That worked for me.

**Comment by JanVasko1** [comment]
Hello all, I tried also what [oplix6417](https://github.com/oplix6417) advice --> it is working but it actually shrinks table to minimal width according to content. I would like to add to this discussion that I'm also looking for this kind of solution. 

My problem is that I have several columns with different content **text**: names, street ... , **integers**: year, day, month. I also need to have table with fixed width and then set different wraplengths for text columns which actually consume more space and int columns let be shorter. This all in the fixed table width. Is there something?
